participant/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,147 @@
+# Translation of the participant app
+# Copyright (C) 2020 Simon Ellmann
+# This file is distributed under the same license as the registration package.
+# Simon Ellmann <simon.ellmann@tum.de>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: 0.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-01-28 17:34+0100\n"
+"PO-Revision-Date: 2020-01-26 00:15+0100\n"
+"Last-Translator: Simon Ellmann <simon.ellmann@tum.de>\n"
+"Language-Team: NOREPLY <noreply@example.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: models/attendance.py:14
+msgid "date"
+msgstr "Datum"
+
+#: models/attendance.py:19 models/attendance.py:20 models/participant.py:110
+msgid "attendance"
+msgstr "Anwesenheit"
+
+#: models/diet.py:13 models/troop.py:19
+msgid "name"
+msgstr "Name"
+
+#: models/diet.py:19 models/participant.py:115
+msgid "diet"
+msgstr "Ernährungsart"
+
+#: models/diet.py:20
+msgid "diets"
+msgstr "Ernährungsarten"
+
+#: models/participant.py:38
+msgid "male"
+msgstr "männlich"
+
+#: models/participant.py:39
+msgid "female"
+msgstr "weiblich"
+
+#: models/participant.py:40
+msgid "diverse"
+msgstr "divers"
+
+#: models/participant.py:50
+msgid "beaver"
+msgstr "Biber"
+
+#: models/participant.py:51
+msgid "cub"
+msgstr "Wölfling"
+
+#: models/participant.py:52
+msgid "scout"
+msgstr "Jungpfadfinder"
+
+#: models/participant.py:53
+msgid "venturer"
+msgstr "Pfadfinder"
+
+#: models/participant.py:54
+msgid "rover"
+msgstr "Rover"
+
+#: models/participant.py:59 models/troop.py:25
+msgid "troop"
+msgstr "Gruppe"
```

Comment:
`Gruppe` wird auch bei `Berechtigungen` verwendet...

Man sollte eins davon ändern glaube ich.

Es könnte hier `Abteilung` heißen zB.

Oder auth.Gruppe könnte `Berechtigung` heißen

Reply:
Hm, `Abteilung` klingt auch doof, vielleicht besser `Gruppierung`? ;)

Reply:
`Truppe` ? (wenn es um Stamm/Bezirk geht)
Oder `Gruppe` lassen, und die Berechtigungsgruppe als `Kategorie` umbennen

Reply:
User.Category (Nutzerkategorie) klingt nicht so schecht, oder?

Reply:
> `Truppe` ? (wenn es um Stamm/Bezirk geht)
> Oder `Gruppe` lassen, und die Berechtigungsgruppe als `Kategorie` umbennen

Für die Diözese wird es auch eine `Gruppierung` geben. Der Begriff passt doch :) Die User-Group umzubennen is bissl komplizierter, dafür gibt's bisher kein Model und wir brauchen vermutlich auch kein bestimmtes.

Reply:
Ok, dann `Gruppierung` :-)